app/services/promo_registrar.rb

```diff
@@ -88,6 +90,18 @@ def youtube_request_body(channel)
     }.to_json
   end
 
+  def twitch_request_body(channel)
+    {
+      "owner_id": @publisher.id,
+      "promo": @promo_id,
+      "channel": channel.channel_id, 
+      "title": channel.publication_title,
+      "channel_type": "twitch",
+      "thumbnail_url": channel.details.thumbnail_url,
+      "description": nil
```

Comment:
we do not have a channel `description` for twitch channels like with youtube channels https://github.com/brave-intl/publishers/blob/1fd27ef14d626fd33afb858237c8c179f8e08bdf/db/schema.rb#L205, but the promo server takes a nullable one (like with youtube). for now we will always give `nil` for the description

Reply:
@nvonpentz you could use `display_name`, which would be something like `MixonicSuperZapper312412` in the wild.

Reply:
@mixonic 
hmm, I think that would be the same as `channel.publication_title` which will be sent to the promo server as the `title`.  If that is correct, then I don't think we want to duplicate the title as the description.